lib/postprocessing/image.js

```diff
@@ -39,16 +39,23 @@ const SUPPORTED_OUTPUT_FMT = {
 // Note: order is important and defines priority for intermediate format - first entry will be most preferred
 const SUPPORTED_INPUT_TYPES = {
     "TIFF": "tif",  // most preferred format as it allows to preserve many features
-    "PNG":  "png",  // lossless over
+    "PNG": "png",  // lossless over
     "JPEG": "jpg",  // ..lossy
-    "GIF":  "gif",  // (should never get here in practice)
-    "BMP":  "bmp"
+    "GIF": "gif",  // (should never get here in practice)
+    "BMP": "bmp",
+    "SVG": "svg"
 };
 const SUPPORTED_INPUT_FORMAT_IMAGEMAGICK = Object.keys(SUPPORTED_INPUT_TYPES);
 const SUPPORTED_INPUT_FMT = Object.values(SUPPORTED_INPUT_TYPES);
+
 const DEFAULT_WORKER_SUPPORTED_OUTPUT_FMT = [
     "png"
 ];
+// list of output formats not supporting alpha channels (transparency)
```

Comment:
FYI - No need to update yet:

In another PR, I named it `FORMAT_SUPPORTS_TRANSPARENCY` (because when people wrote about TIFF, they paid attention to not name it alpha but transparency).
See https://github.com/adobe/asset-compute-sdk/blob/d5095ee4d42296f209b0264228dad6e5e7a575d0/lib/postprocessing/image.js#L54. I'll deal with merging the logic when time to merge https://github.com/adobe/asset-compute-sdk/pull/76 comes.